### PR TITLE
Sprint 2/issue 4/filtering

### PR DESCRIPTION
**Pull request by AnaCarrizales**
Filter tooll has been added

Screenshot from 2024-02-13 17-51-25
Screenshot from 2024-02-13 17-51-45

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -64,4 +64,4 @@ gem 'htmlbeautifier', '~> 1.1', '>= 1.1.1'
 
 gem 'devise', '~> 4.9', '>= 4.9.2'
 
-gem 'image_processing', '~> 1.2'
+gem 'image_processing', '~> 1.2'
```
Comment (juperameza) [inline Gemfile:67]: New linen at the end of the file

File: app/javascript/application.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
+import "@hotwired/turbo-rails"
+import "controllers"
```
Comment (juperameza) [inline app/javascript/application.js:3]: New line

File: app/javascript/controllers/example_controller.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+export default class extends Controller {
+
+  static targets = [ "name" ]
+
+  greet() {
+    const element = this.nameTarget
+    const name = element.value
+    console.log(`hello, ${name}!`)
+  }
+}
```
Comment (juperameza) [inline app/javascript/controllers/example_controller.js:12]: Delete this file if you dont need it

**Review by juperameza — APPROVED** [review]

**Review by MauricioNC — APPROVED** [review]

**Comment by MauricioNC** [comment]
Just attend the comments of jupermeza

**Pull request merged**